chromadb/test/property/test_cross_version_persist.py

```diff
@@ -312,6 +313,7 @@ def test_cycle_versions(
     system = config.System(settings)
     system.start()
     client = ClientCreator.from_system(system)
+    migrate_collections(client)
```

Comment:
shouldn't we only run this if the old version is one we expect to fail?

Reply:
My thought is that any migration we run here must be idempotent in the case there's nothing to be done, and doing this test this way helps ensure that.